packages/app/package.json

```diff
@@ -5,9 +5,9 @@
   "dependencies": {
     "hoist-non-react-statics": "^3.0.1",
     "react": "16.8.6",
-    "react-native": "0.59.5",
-    "react-navigation": "3.9.1",
-    "react-relay": "^4.0.0",
+    "react-native": "0.60.4",
```

Comment:
for react native you need to follow this 

https://react-native-community.github.io/upgrade-helper/

and upgrade all ios and android related code